sdk/ml/azure-ai-ml/azure/ai/ml/constants/_component.py

```diff
@@ -95,5 +95,5 @@ class IOConstants:
         "string": ["default"],
         "boolean": ["default"],
     }
-    GROUP_ATTR_NAME = "__parameter_group__"
+    GROUP_ATTR_NAME = "__dsl_group__"
```

Comment:
make sure we update the mldesigner._utils recommended azure-ai-ml version, when doing this breaking change

Reply:
Ideally we should depend on `azure-ai-ml>=1.2.0`?

Reply:
yes.